dev-aws/kafka-shared-msk/customer-proposition/__env.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
```

Comment:
This could just be a symlink to the one in the parent folder. It would be easier to control it this way.